spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/person.rb'
+
+  describe Person do
+#     books = [{title: 'Managment101', author: 'Rich' }, { title: 'Cooking101', author: 'Fat'}]
+
+#     it 'constract of empty array at start' do
+#    expect(subject.portfolio).to eq []
+# end
+#
+#  context 'adding book to portfolio' do
+#    before { allow(library).to receive(:check_out).and_return({ }) }
+#
+#    it 'searched by :title' do
+#      subject.rent(library, title: 'Cooking101')
+#      expect(subject.portfolio).to eq [{ title: 'Cooking101', author: 'Fat' }]
+#    end
+#
+#    it 'searched by :author' do
+#      subject.rent(library, author: 'Rich')
+#      expect(subject.portfolio).to eq [{ title: 'Managment101', author: 'Rich' }]
+#    end
+#  end
+end
```

Comment:
Lint/BlockAlignment: `end` at 23, 0 is not aligned with `describe Person do` at 3, 2.